src/driver/mongodb.rs

```diff
@@ -139,9 +105,8 @@ impl From<Bson> for Value {
       Bson::JavaScriptCode(value) => Value::String(value),
       Bson::JavaScriptCodeWithScope(value, _) => Value::String(value),
       Bson::I32(value) => Value::I64(i64::from(value)),
-      Bson::I64(value) => Value::I64(i64::from(value)),
+      Bson::I64(value) => Value::I64(value),
       Bson::TimeStamp(value) => Value::I64(i64::from(value)),
-      // TODO: Actual transformation of binary type.
       Bson::Binary(_, _) => Value::Null,
```

Comment:
Do you know a good way to transform this variant to JSON? The docs for this type are [here](http://mongodb-labs.github.io/mongo-rust-driver-prototype/bson/enum.Bson.html)